husky: command not found

The latest version (v2.0.1) introduced running `husky` in the postinstall script and it breaks installing the package:
```
npm install @opensea/seaport-js
npm ERR! code 127
npm ERR! path /Users/user/research/seaport-example/node_modules/@opensea/seaport-js
npm ERR! command failed
npm ERR! command sh -c -- husky install
npm ERR! sh: husky: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/user/.npm/_logs/2023-06-04T07_09_32_448Z-debug-0.log
```

More info from the debug log:
```
184 http fetch GET 200 https://registry.npmjs.org/@opensea/seaport-js/-/seaport-js-2.0.1.tgz 1784ms (cache miss)
185 timing reifyNode:node_modules/@opensea/seaport-js Completed in 1802ms
186 timing reify:unpack Completed in 1804ms
187 timing reify:unretire Completed in 0ms
188 timing build:queue Completed in 2ms
189 info run @opensea/seaport-js@2.0.1 postinstall node_modules/@opensea/seaport-js husky install
190 info run @opensea/seaport-js@2.0.1 postinstall { code: 127, signal: null }
191 timing reify:rollback:createSparse Completed in 1038ms
192 timing reify:rollback:retireShallow Completed in 1ms
193 timing command:install Completed in 13900ms
194 verbose stack Error: command failed
194 verbose stack     at ChildProcess.<anonymous> (/Users/user/.nvm/versions/node/v16.20.0/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/lib/index.js:63:27)
194 verbose stack     at ChildProcess.emit (node:events:513:28)
194 verbose stack     at maybeClose (node:internal/child_process:1100:16)
194 verbose stack     at Socket.<anonymous> (node:internal/child_process:458:11)
194 verbose stack     at Socket.emit (node:events:513:28)
194 verbose stack     at Pipe.<anonymous> (node:net:301:12)
195 verbose pkgid @opensea/seaport-js@2.0.1
196 verbose cwd /Users/user/research/seaport-example
197 verbose Darwin 22.5.0
198 verbose node v16.20.0
199 verbose npm  v8.19.4
200 error code 127
201 error path /Users/user/research/seaport-example/node_modules/@opensea/seaport-js
202 error command failed
203 error command sh -c -- husky install
204 error sh: husky: command not found
205 verbose exit 127
206 timing npm Completed in 13961ms
207 verbose unfinished npm timer reify 1685862572507
208 verbose unfinished npm timer reify:build 1685862585332
209 verbose unfinished npm timer build 1685862585333
210 verbose unfinished npm timer build:deps 1685862585333
211 verbose unfinished npm timer build:run:postinstall 1685862585335
212 verbose unfinished npm timer build:run:postinstall:node_modules/@opensea/seaport-js 1685862585335
213 verbose code 127
214 error A complete log of this run can be found in:
214 error     /Users/user/.npm/_logs/2023-06-04T07_09_32_448Z-debug-0.log
```

I see it's installed as a dev dependency, so I'm guessing there are two ways to solve the issue:
1. Either add `husky` as a non-dev dependency
2. Or remove/rewrite the [postinstall](https://github.com/ProjectOpenSea/seaport-js/blob/main/package.json#LL31C27-L31C34) script to not use `husky` (perhaps another [npm script](https://docs.npmjs.com/cli/v9/using-npm/scripts) type is a better fit?)

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@opensea/seaport-js",
-  "version": "2.0.1",
+  "version": "2.0.2",
   "description": "[Seaport](https://github.com/ProjectOpenSea/seaport) is a new marketplace protocol for safely and efficiently buying and selling NFTs. This is a JavaScript library intended to make interfacing with the contract reasonable and easy.",
   "license": "MIT",
   "author": "OpenSea Developers",

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
     "check-types:incremental": "npm run check-types --incremental",
     "coverage": "c8 npm run test",
     "eslint:check": "eslint . --max-warnings 0 --ext .js,.jsx,.ts,.tsx",
-    "postinstall": "husky install",
+    "postinstall": "husky install || true",
     "lint": "concurrently \"npm run check-types\" \"npm run eslint:check\" \"npm run prettier:check\" \"npm run prettier:check:package.json\"",
     "prepare": "npm run build",
     "prettier:check": "prettier --check .",

